### PR TITLE
don't forget to store log-level arg

### PR DESCRIPTION
**Pull request by minrk**
this must be explicit for custom actions

### CODE REVIEW

**Comment by martinal** [comment]
The timestamp bug the tests fail with here is fixed in my current PR (commit 64f69049).

**Comment by minrk** [comment]
@martinal is your PR open? There are several other small bugs found by setting the debug level. I've fixed them locally, but can wait for your PR and rebase.

File: nbdime/prettyprint.py [hunk 1]
```diff
@@ -305,7 +305,7 @@ def pretty_print_metadata(md, known_keys, prefix="", out=sys.stdout):
 
 def pretty_print_output(i, output, prefix="", out=sys.stdout):
     oprefix = prefix+IND
-    t = output.output_type
+    t = output['output_type']
```
Comment (martinal) [inline nbdime/prettyprint.py]: Why? I've been adding NotebookNode elsewhere to make sure we can use . notation. If you don't want that a lot of prettyprint.py needs to be updated.

File: nbdime/nbmergeapp.py [hunk 2]
```diff
@@ -56,8 +56,8 @@ def main_merge(args):
     else:
         if conflicted:
             out = io.StringIO()
-            pretty_print_merge_decisions(b, conflicted, stream=out)
-            nbdime.log.warn("Conflicts: %s", out.getvalue())
+            pretty_print_merge_decisions(b, conflicted, out=out)
+            nbdime.log.warning("Conflicts: %s", out.getvalue())
```
Comment (martinal) [inline nbdime/nbmergeapp.py]: Also fixed in my branch...

File: nbdime/merging/autoresolve.py [hunk 3]
```diff
@@ -129,11 +131,11 @@ def make_inline_outputs_value(value, le, re,
...
     # Note: This is very notebook specific while the rest of this file is more generic
     newvalue = (
-        [{"output_type": "stream", "name": "stderr", "text": ["<"*7 + local_title]}]
+        [new_output('stream', name='stderr', text="<" * 7 + local_title)]
         + lvalue
-        + [{"output_type": "stream", "name": "stderr", "text": ["="*7]}]
+        + [new_output('stream', name='stderr', text="=" * 7)]
         + rvalue
-        + [{"output_type": "stream", "name": "stderr", "text": ["<"*7 + remote_title]}]
+        + [new_output('stream', name='stderr', text=">" * 7 + remote_title)]
```
Comment (martinal) [inline nbdime/merging/autoresolve.py]: Also in my branch...

**Comment by martinal** [comment]
@minrk You can merge my PR now if you want, and just drop the non-log changes from this one then I'll merge it ASAP.

**Comment by martinal** [comment]
I've cherry-picked the log fix to my PR "Various fixes".

**Pull request closed without merging**